--- main.orig	2018-03-14 17:51:07.602502694 -0700
+++ main.c	2018-03-14 17:51:19.106848130 -0700
@@ -735,12 +735,6 @@ Py_Main(int argc, wchar_t **argv)
     if ((Py_InspectFlag || (command == NULL && filename == NULL && module == NULL)) &&
         isatty(fileno(stdin)) &&
         !Py_IsolatedFlag) {
-        PyObject *v;
-        v = PyImport_ImportModule("readline");
-        if (v == NULL)
-            PyErr_Clear();
-        else
-            Py_DECREF(v);
     }
 
     if (command) {
